libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/PreOreoAudioFocusDelegate.kt

```diff
@@ -8,12 +8,12 @@ internal class PreOreoAudioFocusDelegate(
     private val options: VoiceInstructionsPlayerOptions
 ) : AudioFocusDelegate {
 
-    override fun requestFocus() {
-        audioManager.requestAudioFocus(
+    override fun requestFocus(): Boolean {
+        return audioManager.requestAudioFocus(
             null,
             AudioManager.STREAM_MUSIC,
```

Comment:
See comment above re: stream type

Should we use the stream type provided when requesting the audio focus? 🤔